src/plugins/Network.js

```diff
@@ -122,37 +122,27 @@ module.exports = class Network extends OverlayPlugin {
             this.profilesForDropdownSignalStrength.find((ms) => ms.value === signalStrength[1]),
         );
 
-        this.updateDetail(
-            'downSpeed',
-            downSpeed[2] + ' b/s',
-            downSpeed[1] === 'disabled' || !this.mobileDataSwitch.getState(),
-        );
-        this.updateDetail(
-            'upSpeed',
-            upSpeed[2] + ' b/s',
-            upSpeed[1] === 'disabled' || !this.mobileDataSwitch.getState(),
-        );
-        this.updateDetail(
-            'downDelay',
-            downDelay[2] + ' s',
-            downDelay[1] === 'disabled' || !this.mobileDataSwitch.getState(),
-        );
-        this.updateDetail('upDelay', upDelay[2] + ' s', upDelay[1] === 'disabled' || !this.mobileDataSwitch.getState());
+        // Disable signal strength dropdown and details if network type is none
```

Comment:
Could add to the commit message something that indicates that we now disable detail attributes if the network type is 'Unlimited' ("none").